index.html

```diff
@@ -9,4 +9,3 @@
 <body>
     
```

Comment:
- [ ] I kindly suggest that you implement the HTML and CSS code to bring this template or your selected template to life. 
- [ ] Kindly ensure that all you implement all project functionalities. 

Project requirements
For this project, you should use the [HTML&CSS linter](https://github.com/microverseinc/linters-config/tree/master/html-css)

## The 2 sections you need to build for this project are:

the toolbar (or header):
![toolbar-mobile](https://user-images.githubusercontent.com/52934128/198863438-1198ba3b-6e98-40f4-9130-c3089b9268d2.png)

the headline section (right after the header):

![headline-mobile (1)](https://user-images.githubusercontent.com/52934128/198863475-dd2b5cff-d558-4860-a2ef-19add5fc05b3.png)